### DISCUSSION TITLE
Detect inner element

### DISCUSSION THREAD

**Discussion by creative-eye-studio** [idiotWu/smooth-scrollbar#426] [Q&A]
Hello everyone

I use smooth-scrollbar with VueJS. and I would like to know how to detect scrollTop of an element inner to show him with AOS.

Thanks for your help :)

**Comment by dolphin-wood** [comment]
Try:

```js
elem.getBoundingClientRect().top - scrollbar.containerEl.getBoundingClientRect().top
```
